doc/install.rst

```diff
@@ -0,0 +1,148 @@
+.. Copyright (c) 2008-2020 OpenShot Studios, LLC
+ (http://www.openshotstudios.com). This file is part of
+ OpenShot Video Editor (http://www.openshot.org), an open-source project
+ dedicated to delivering high quality video editing and animation solutions
+ to the world.
+
+.. OpenShot Video Editor is free software: you can redistribute it and/or 
+ modify it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
+ along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+
+.. Edited September 27, 2020 - [USA]TechDude
+
+Install |ops| OpenShot
+-----------------------
+
+OpenShot™ is available for the following 64-bit operating systems: Microsoft 
+Windows®, Linux® (most distributions), and Apple® macOS®.
+
+|win| Windows®
+^^^^^^^^^^^^^^
+
+Download the Windows installer executable from the official OpenShot |Link|_.
+
+.. image:: images/install-win1.png
+
+Choose to save the file to your system so you can launch the installer after 
+the download completes.
+
+.. image:: images/install-win2.png
+
+Once downloaded, launch the EXE file, and follow the directions on screen. 
+Choose the language that you want to use if it is not already highlighted in 
+the pull-down menu. 
+
+.. image:: images/install-win3.png
+
+Read the License Agreement and select the radio button in front of "I accept 
+the agreement" if you agree to the terms.
+
+.. image:: images/install-win4.png
+
+Choose where to install OpenShot or use the default path provided.
+
+.. image:: images/install-win5.png
+
+Select any additional shortcuts that you want setup to create when it installs 
+the application.
+
+.. image:: images/install-win6.png
+
+Review the options for installing OpenShot.  Click the back button to make 
+changes or click Install to install OpenShot on your system.
+
+.. image:: images/install-win7.png
+
+When Setup completes, you can choose to launch the program by leaving the check 
+in the box next to "Launch OpenShot Video Editor" and clicking "Finish".
+
+.. image:: images/install-win8.png
+
+Setup created an icon in your Start Menu from where you can launch the program.
+
+.. image:: images/install-win9.png
+
+|lin| Linux®
```

Comment:
We have a new install method for Chrome OS now (although it uses the same AppImage files), the installation is located on https://www.openshot.org/download/ (Click Chrome OS on the bottom). All 4 OSes should be auto-detected using useragent on the browser, and will always show you the correct version of the download page.